src/reducers/paymentMethods.js

```diff
@@ -82,11 +85,27 @@ const wyre = (state, action) => {
           ...action.payload.config
         },
       };
+    case 'GET_ACTIVE_TRANSACTION':
+        return {
+          ...state,
+          ui: isFetching(state.ui, actionToUi(action), false),
```

Comment:
when we request rates we dispatch GET_ACTIVE_TRANSACTION and in that phase isFetching should be true until we get the rates and that is GET_ACTIVE_TRANSACTION_RESPONSE action then we should set it to false